main.go

```diff
@@ -59,9 +62,41 @@ func validateRequirements(dataplaneBin, haproxyBin string) error {
 	return nil
 }
 
+func validateLogAddress(logAddress string) error {
+	fi, err := os.Stat(logAddress)
+	if err != nil {
+		// maybe it's a syslog host[:port]
+		if len(strings.Split(logAddress, ":")) == 1 {
```

Comment:
Use `net.SplitHostPort()` here instead to properly handle IPv6

Reply:
Thanks! But it turned out to be complicated to check if the error out of `net.SplitHostPort()` is "missing port" so I did something else, I think that it's even more elegant. :) Can you please check this code now?

I also added one more case that HAproxy supports - file descriptor/stdout/stderr.

Reply:
Is the current validation code ok for you, @ShimmerGlass ?